src/test/java/com/bunq/sdk/http/PaginationTest.java

```diff
@@ -0,0 +1,143 @@
+package com.bunq.sdk.http;
+
+import com.bunq.sdk.exception.BunqException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * Tests:
+ * Pagination
+ */
+public class PaginationTest {
+
+  /**
+   * Values of pagination properties for testing.
+   */
+  private static final int PAGINATION_OLDER_ID_CUSTOM = 1;
+  private static final int PAGINATION_NEWER_ID_CUSTOM = 2;
+  private static final int PAGINATION_FUTURE_ID_CUSTOM = 3;
+  private static final int PAGINATION_COUNT_CUSTOM = 5;
+
+  @Rule
+  public final ExpectedException exception = ExpectedException.none();
+
+  @Test
+  public void getUrlParamsCountOnlyTest() {
```

Comment:
This comments count for this entire file. In the other tests there are comment docs on each test briefly explaining what this test is actually testing. Don't we want to keep doing this for consistency ?

Reply:
Explained above, in a comment for `PaginationScenarioTest`